waspc/test/Analyzer/Evaluation/EvaluationErrorTest.hs

```diff
@@ -1,27 +1,80 @@
 module Analyzer.Evaluation.EvaluationErrorTest where
 
 import Analyzer.TestUtil (ctx)
+import Data.List (intercalate)
 import Test.Tasty.Hspec
 import Wasp.Analyzer.Evaluator.EvaluationError
+import Wasp.Analyzer.Parser.Ctx
 import Wasp.Analyzer.Type (Type (..))
 
+ctx1 :: Ctx
+ctx1 = ctx (1, 4) (2, 5)
+
+nestEvalErrors :: EvaluationError -> [EvalErrorCtx] -> EvaluationError
+nestEvalErrors bottomErr wrapperErrs = foldr wrap bottomErr wrapperErrs
+  where
+    wrap evalErrorCtx = mkEvaluationError ctx1 . WithEvalErrorCtx evalErrorCtx
+
 spec_EvaluationError :: Spec
 spec_EvaluationError = do
   describe "Analyzer.Evaluator.EvaluationError" $ do
     describe "getErrorMessageAndCtx works correctly for" $ do
+      let expectedTypeError = mkEvaluationError ctx1 $ ExpectedType NumberType StringType
+      let expectedTypeErrorMsg = "Expected type: number\nActual type:   string"
```

Comment:
I extracted these two because I reused them in multiple tests, is that a normal thing to do (note that their original test is now only an equality check, the code is actually executed outside)? @Martinsos